### DISCUSSION TITLE
SKR V1.4 Turbo

### DISCUSSION THREAD

**Discussion by scaramonga** [Jyers/Marlin#987] [Q&A] [answered]
Yes or no with this firmware on Ender 3 V2?  2209 steppers.  Is screen also OK for use, or will I need another?

**Comment by Jyers** [comment, marked as answer]
Everything should work, but screen will almost definitely need to be repinned. See #952

**Comment by scaramonga** [comment]
Nah I'll just get a TFT35 then, thanks :)  My eyes and soldering skills have long gone lol :D  A bottle of Whisky can stop the shakes, but then my eyes are more blurred ;)

**Reply by creativeG911** [reply]
Do you mind to share the whole pack with us Who love jyers filware and bought the board with the trt?
